### PR TITLE
Crates - 3rd times a charm

### LINKED ISSUE #553: Define cargo crate prototypes
The CargoConsole doesn't have any crates currently, this needs to be fixed. Add crates and fill them with stuff.

### PR DESCRIPTION
**Pull request by SweptWasTaken**
3rd and hopefully final PR for this. Been a PR in some form or another since April.

A lot of the stuff for the actual crates isn't implemented yet, so I either didn't include them or commented them out. I took most of them from tgstation's current code.

- [x] Changed plasma crate to phoron
- [x] Fix crate icons
- [x] Add the rest of the crates
- [x] Make it so you can weld crates shut
- [x] Make it so you can anchor crates
- [x] Add tgcrates content
- [x] Add secure crates that can be "locked" (Access levels for secure storage will be in a different PR)
- [x] Add crates to cargo console

Will close #553 and old sprites will be added to the rsi-dump.

Just a reminder, these crates are priced based on tgstation's code. They are probably horribly unbalanced for SS14, so i'll definitely be changing these around at some point.

### CODE REVIEW

File: Resources/Prototypes/Entities/Objects/Fun/toys.yml [hunk 1]
```diff
@@ -60,6 +60,21 @@
...
+  components:
+  - type: Toys
+  - type: LoopingSound
+  - type: Sprite
+    sprite: Objects/Fun/toys.rsi
+    state: giantsmugplushie
+  - type: ItemCooldown
+  - type: UseDelay
+    delay: 1.0
+
```
Comment (metalgearsloth) [inline Resources/Prototypes/Entities/Objects/Fun/toys.yml:77]: highly approve

File: Resources/Prototypes/Entities/Constructible/Storage/Crates/Filled/crate_science.yml [hunk 2]
```diff
@@ -0,0 +1 @@
+# Bro what do I PUT here
```
Comment (metalgearsloth) [inline Resources/Prototypes/Entities/Constructible/Storage/Crates/Filled/crate_science.yml:1]: idk lol

**Review by metalgearsloth — APPROVED** [review]
Fix build

File: Resources/Prototypes/Entities/Objects/Fun/toys.yml (hunk 1 above)
Comment (ike709) [reply Resources/Prototypes/Entities/Objects/Fun/toys.yml:77]: `It's` should be `Its` though.

File: Resources/Prototypes/Entities/Constructible/Power/cargo_console.yml [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+      - cargo.securitynonlethal
+      - cargo.hydroponics
+      - cargo.hydroponicsseeds
+      - cargo.hydroponicsseedsexotic
+      - cargo.servicejanitorial
+      - cargo.servicelightsreplacement
+      - cargo.funplushies
+      - cargo.funartsupplies
+      - cargo.funinstruments
+      -
```
Comment (Zumorica) [inline Resources/Prototypes/Entities/Constructible/Power/cargo_console.yml:37]: oops?
```suggestion
```

File: Resources/Prototypes/Entities/Objects/Fun/toys.yml [hunk 4]
```diff
@@ -60,6 +60,21 @@
   - type: UseDelay
     delay: 1.0
 
+- type: entity
+  name: giant smug plushie
+  parent: BaseItem
+  id: PlushieSmug
+  description: It's smug aura compells you to perform surgery with a slightly higher degree of skill.
```
Comment (Zumorica) [inline Resources/Prototypes/Entities/Objects/Fun/toys.yml:67]:
```suggestion
  description: Its smug aura compells you to perform surgery with a slightly higher degree of skill.
```

**Review by Zumorica — CHANGES_REQUESTED** [review]

**Comment by Zumorica** [comment]
I fixed everything wrong with this PR, I'm merging this.
Needs more work but that can be done in the future, and this is already a good starting point anyway.

**Pull request merged**